pkg/scaffold/v1/webhook/webhook_test.go

```diff
@@ -39,39 +39,64 @@ var _ = Describe("Webhook", func() {
 	serverName := "default"
 	inputs := []*webhookTestcase{
 		{
-			Resource: resource.Resource{Group: "crew", Version: "v1", Kind: "FirstMate", Namespaced: true, CreateExampleReconcileBody: true},
+			Resource: resource.Resource{
+				Group:      "crew",
+				Version:    "v1",
+				Kind:       "FirstMate",
+				Namespaced: true,
+				CreateExampleReconcileBody: true},
```

Comment:
IMO it's better to move the closed bracket `}` to its own line.

Reply:
And add a comma at the end of `CreateExampleReconcileBody: true,`

Reply:
Done.